package.json

```
@@ -19,7 +20,7 @@
     "mysql": "^2.17.1",
     "notify-queue": "0.0.5",
     "passport": "^0.4.0",
-    "passport-github": "1.1.0",
+    "passport-github2": "^0.1.12",
```

I looked through [the diff](https://github.com/jaredhanson/passport-github/compare/master...cfsghost:master) between `passport-github` and `passport-github2`, and it looks totally reasonable. I'm not quite sure how hard we want to version lock this; I'm nervous about the fact that this is one semi-random guy's fork.

---

(when I say "how hard we want to version lock this", I mean "do we want to use a version like
```suggestion
    "passport-github2": "0.1.12",
```
which won't allow any other version to be used (although I don't think it prevents malicious uploads over `0.1.12` on NPM).

---

The package looks pretty small, actually, looking over at https://github.com/cfsghost/passport-github/tree/master/lib. I wonder if we would want to fork passport-github2, take a little time to understand it, and install from our fork.